data/Nigeria/Senate/sources/instructions.json

```diff
@@ -21,6 +21,21 @@
         "from": "group-wikidata",
         "source": "manual/group_wikidata.csv"
       }
-    }
+    },
+    {
+      "file": "morph/shineyoureye.csv",
+      "create": {
+        "from": "morph",
+        "scraper": "struan/nigeria-shineyoureye",
+        "query": "SELECT *, NULL as position FROM data WHERE position = 'Senator'"
```

Comment:
The immediate use for us is so that we can make sure that the old URLs for people on the Pombola-based shineyoureye.org work on the new static site without any redirection.

I do care much more about this being merged soon than I do about the name of the identifier scheme, though, so feel free to ignore this.